tests/crypto/crypto.cpp

```diff
@@ -33,7 +33,7 @@
 #include "crypto-tests.h"
 
 bool check_scalar(const crypto::ec_scalar &scalar) {
-  return crypto::sc_check(crypto::operator &(scalar)) == 0;
+  return sc_check(crypto::operator &(scalar)) == 0;
```

Comment:
Why ?

Reply:
good question. A mod/unmod during my G++ fight

reverted

Reply:
because of:

    /home/cme/Projects/Git/Monero/monero-cslashm/tests/crypto/crypto.cpp:36:18: error: ‘sc_check’ is not a member of ‘crypto’
       return crypto::sc_check(crypto::operator &(scalar)) == 0;

Moreover sc_check is defined in src/crypto/crypto-ops.c outside any namspace. So I dont understand this `crypto::`

Do I miss something?

Reply:
Well, it was compiling fine before, no ?

Reply:
To be honest I do not remember as I had dis-activated test compilation for a long period.

Reply:
That needs to be worked out. It should not be needed unless something broke.

Reply:
see below